postgres/datadog_checks/postgres/statements.py

```diff
@@ -164,6 +164,18 @@ def collect_per_statement_metrics(self, db, db_version, tags):
             self._log.exception('Unable to collect statement metrics due to an error')
             return []
 
+    def _load_pg_stat_statements_safe(self, db, tags):
+        try:
+            self._load_pg_stat_statements(db)
+        except psycopg2.errors.ObjectNotInPrerequisiteState as e:
+            if 'pg_stat_statements must be loaded via shared_preload_librarie' in str(e):
+                self._check.count("postgresql.setup.error", 1, tags=tags + ["reason:pg_stat_statements_not_enabled"])
```

Comment:
It would be good to include the `db` tag. If I'm reading correctly on L141, we drop that tag for some reason when passing in here.

Reply:
That's a good point. I updated this to pass in the full tags and kept a `tags_no_db` for the existing code.